docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -519,6 +519,28 @@ These rules govern the privileges granted on specific tables.
 
 * ``filter_environment`` (optional): environment use during filter evaluation.
 
+.. note::
+
+    Privileges ``OWNERSHIP`` are required when following actions are taken on the table::
```

Comment:
``OWNERSHIP`` privilege is required for performing the following operations

Reply:
Instead of separate lists for table and view, we should have separate lists for:
operations requiring schema ownership (isSchemaOwner in FileBasedAccessControl)
operations requiring table ownership (checkTablePermission(context, tableName, OWNERSHIP) in FileBasedAccessControl)